thirdparty/libbacktrace/config.h

```diff
@@ -48,7 +52,7 @@
 /* #undef HAVE_KERN_PROC_ARGS */
 
 /* Define if -llzma is available. */
-#define HAVE_LIBLZMA 1
+/* #undef HAVE_LIBLZMA */
```

Comment:
This was likely just what happened to be on bruvzg's system last time he synced.

I generated this config with:
```
./configure --host x86_64-w64-mingw32 --target x86_64-w64-mingw32 --build x86_64-w64-mingw32
```

so it should be a bit more Windows compatible by default.

Either way `HAVE_LIBLZMA` isn't used by any of the files we include.